drivers/lvutil.py

```diff
@@ -972,7 +972,13 @@ def _lvmBugCleanup(path):
             except util.CommandException, e:
                 if i < LVM_FAIL_RETRIES - 1:
                     util.SMlog("Failed on try %d, retrying" % i)
-                    time.sleep(1)
+                    cmd = [CMD_DMSETUP, "status", mapperDevice]
+                    try:
+                        util.pread(cmd, expect_rc=1)
+                        util.SMlog("_lvmBugCleanup: dm device %s removed" % mapperDevice)
```

Comment:
Line too long.
Please, use format() for formatting strings instead of %s

Use any pep8 compliant program to check syntax (I suggest pep8radius).